e2e/example_client/example_client_test.go

```diff
@@ -0,0 +1,27 @@
+package example_client
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestRestartProcess(t *testing.T) {
+	client := StartLiveReload(t)
+	defer client.Cleanup()
+	client.Kill(t)
+
+	// Wait for the server to kill itself (it has a 1 second delay)
+	time.Sleep(time.Second * 2)
```

Comment:
Would it be better to in `client.Kill()` try to get the `/config` url over and over again until a connection error? Then there wouldn't need to be a sleep in here

Reply:
Done